src/vmm/src/builder.rs

```diff
@@ -211,7 +210,7 @@ impl VmmEventsObserver for SerialStdin {
 /// independent functions in this module instead of calling this recipe.
 pub fn build_microvm(
     vm_resources: &super::resources::VmResources,
-    epoll_context: &mut EpollContext,
+    _epoll_context: &mut EpollContext,
```

Comment:
vsock is the last user of `EpollContext`, would you like to add a commit to this PR to completely remove the `EpollContext`?

Reply:
Done!